src/main/java/gyro/google/compute/ImageResource.java

```diff
@@ -451,8 +451,7 @@ protected void doCreate(GyroUI ui, State state) throws Exception {
         image.set("storageLocations", getStorageLocations());
 
         Operation operation = client.images().insert(getProjectId(), image).execute();
-        // Images are slow to complete so wait max of 3 minutes for completion
-        waitForCompletion(client, operation, 180000);
+        waitForCompletion(client, operation);
```

Comment:
Based on the other review, this should call the custom waitForCompletion method and specify the 3 minute wait, as this is an exceptional case. Also keep the comment as that proves this is an exceptional case.